packages/myst-cli/src/transforms/outputs.ts

```diff
@@ -38,12 +38,14 @@ export async function transformOutputsToCache(
   if (!outputs.length) return;
   const cache = castSession(session);
   await Promise.all(
-    outputs.map(async (output) => {
-      output.data = await minifyCellOutput(output.data as IOutput[], cache.$outputs, {
-        computeHash,
-        maxCharacters: opts?.minifyMaxCharacters,
-      });
-    }),
+    outputs
+      .filter((output) => output.visibility !== 'remove')
```

Comment:
This will break thebe. Can we do it in the render instead?

Similar to what we already do for the blocks:
https://github.com/executablebooks/mystmd/blob/main/packages/myst-to-typst/src/index.ts#L108

I think that just needs to be done for outputs/code.

Reply:
Oh wait. This is outputs!